extensions/volcano/generator-v1beta1/.gitignore

```diff
@@ -0,0 +1 @@
+/vendor/
```

Comment:
Is it possible to put this in root level [`.gitignore`](https://github.com/fabric8io/kubernetes-client/blob/master/.gitignore)?

Reply:
I think so, hadn't seen that we were doing that for other parts of the generated stuff.  It would be good to capture a task for this.